storage/storage.go

```diff
@@ -73,9 +92,16 @@ type Store interface {
 	GetAllStats(ctx context.Context, queues []string) (*Stats, error)
 
 	// Heartbeat
+	GetActiveHeartbeatIDs(ctx context.Context) ([]string, error)
+	GetHeartbeat(ctx context.Context, heartbeatID string) (*Heartbeat, error)
```

Comment:
How about combining these two into `GetAllHeartbeats()`?

Reply:
Updated in 5946b8ef.

Reply:
Thank you for updating this.